__tests__/index.test.js

```diff
@@ -154,3 +154,57 @@ describe('Bad routes', () => {
     });
 });
 
+describe('POST /api/reviews/:review_id/comments', () => {
+    test('GET 201: accepts request body object with username and body and responds with the posted comment', () => {
+        const newComment = {
+            username: 'mallionaire',
+            body: 'I enjoy playing this game every time and wit the usual suspects'
+        };
+        return request(app)
+            .post('/api/reviews/2/comments')
+            .send(newComment)
+            .expect(201)
+            .then(response => {
+                expect(response.body.insertedComment).toMatchObject({
+                    comment_id: 7,
+                    body: 'I enjoy playing this game every time and wit the usual suspects',
+                    review_id: 2,
+                    author: 'mallionaire',
+                    votes: 0,
+                    created_at: expect.any(String)
+                });
+            });
+
+    });
```

Comment:
Add another 201 test that ignores unnecessary properties on the newComment body e.g. send {username: "validuser", body: "validbody", unnecessary: "property"} - expect the unnecessary property to be ignored and the username/body to be posted successfully.